packages/components/molecules/f-rating/src/components/_tests/Rating.test.js

```diff
@@ -157,5 +158,97 @@ describe('Rating', () => {
                 expect(validator(6)).toBe(false);
             });
         });
+
+        describe('`ratingDisplayType`', () => {
+            it.each(VALID_STAR_RATING_DISPLAY_TYPE)('should allow correct value `%s`', displayType => {
+                // Act
+                const { validator } = VRating.props.ratingDisplayType;
+
+                // Assert
+                expect(validator(displayType)).toBe(true);
+            });
+
+            describe('when `ratingDisplayType` is truthy', () => {
```

Comment:
Does it require a test when the `ratingDisplayType` is `falsy` ?

Reply:
Fixed.